sw/airborne/firmwares/rotorcraft/stabilization/wls/qr_solve.c

```diff
@@ -0,0 +1,1616 @@
+/*
+ * This is part of the qr_solve library from John Burkardt.
```

Comment:
Maybe this part should go to the math library if it is really generic.
Also we already have some tools for matrix decomposition (including QR) or linear system solving (using SVD). Not sure if it is useful here, nor which one is the most efficient.